flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/testutils/junit/SharedObjects.java

```diff
@@ -75,7 +75,7 @@
      */
     private final int id;
     /** All registered objects for the current test case. The objects are purged upon completion. */
-    private final transient Map<SharedReference<?>, Object> objects = new ConcurrentHashMap<>();
+    private final Map<SharedReference<?>, Object> objects = new ConcurrentHashMap<>();
```

Comment:
Nit: Make a separate commit for this change? It is not obvious why this change is needed.

Reply:
I completely removed it. It's just a warning currently. I thought of using SharedObjects and then it would make sense to hotfix it here. But now I'll do it next time I touch SharedObjects.